.github/workflows/tools_release_tag.yml

```diff
@@ -21,6 +21,8 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
do we need to give "actions: read" permission?

Reply:
I understand `write` permission should imply `read`, so we don't need to specify `read, write` here.

Reply:
unresolve this to make it a blocker, we still need to confirm something with this GH change

Reply:
resolve, we have test that current permission is sufficient.